css/app/calendarlist.css

```diff
@@ -212,6 +212,16 @@
 	box-sizing: border-box;
 }
 
+#app-navigation .app-navigation-list-item {
+	max-height: 1000px;
```

Comment:
This is needed for the transition. You can't define a transition from height:0 to height:auto